### PR TITLE
Added PSR-6 to Illuminate adapter

### PR DESCRIPTION
**Pull request by florianv**
| Question      | Answer
| ------------- | ------
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | https://github.com/php-cache/issues/issues/81
| License       | MIT
| Doc PR        | 

### Description

This PR adds a PSR-6 to Illuminate (Laravel Cache) adapter.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -35,7 +35,7 @@
         "cache/integration-tests": "^0.16",
         "predis/predis":           "^1.0",
         "symfony/cache":           "dev-master",
-        "illuminate/cache":        "^5.0"
+        "illuminate/cache":        "^5.4"
```
Comment (florianv) [inline composer.json]: This version bump is needed so we can receive cache flush feedback https://github.com/illuminate/cache/commit/8c71f266e9206354c8957bac4a4e571fa64ed4eb

Comment (florianv) [inline composer.json]: We should skip the PHP 5.5 build for this package as Illuminate 5.4 requires PHP 5.6+. Do you have an idea for this @Nyholm ?

Comment (aequasi) [inline composer.json]: I cant remember, is there a reason we still support php 5.5 @Nyholm?

Comment (Nyholm) [inline composer.json]: No. We just have not had a valid reason to remove 5.5. This is a valid reason.

**Comment by florianv** [comment]
Done @aequasi @Nyholm

**Comment by Nyholm** [comment]
Thank you @florianv and @prisis. 

Im going to merge this and set up the package. After that it will be easer to test and find possible issues.

**Comment by Nyholm** [comment]
https://github.com/php-cache/illuminate-adapter

I added packagist, travis and scrutinizer.

**Pull request merged**